settings.gradle.kts

```
@@ -0,0 +1,8 @@
+rootProject.name = "HelloWorldExtension"
```

```suggestion
rootProject.name = "hivemq-hello-world-extension"
```
Camelcase is not the convention for project names and all our other extensions follow the other pattern